cache.ts

```diff
@@ -21,6 +21,13 @@ export class FetchCacher {
     checksum?: string,
   ): Promise<LoadResponse | undefined> => {
     const url = new URL(specifier);
-    return this.#fileFetcher.fetchOnce(url, { cacheSetting, checksum });
+    return this.#fileFetcher.fetchOnce(url, { cacheSetting, checksum })
+      .catch((e) => {
+        if (e instanceof Deno.errors.NotFound) {
+          return undefined;
+        }
+
+        throw new Error("FetchCacher#load failed", { cause: e });
```

Comment:
I think this is too much information to add. Maybe we should just rethrow?

Reply:
Sounds good 👍  [0891e9f](https://github.com/denoland/deno_cache_dir/pull/64/commits/0891e9fd5dcb25229f8878397bc89c4ea1b502e9)